requirements.txt

```diff
@@ -0,0 +1,2 @@
+# generated from manifests external_dependencies
+openupgrade
```

Comment:
```suggestion
openupgradelib
```
![image](https://user-images.githubusercontent.com/1033124/205307685-dadfa093-09f5-4903-b48b-f913ea34d800.png)

Reply:
Thanks @StefanRijnhart for your review. I corrected it

Reply:
Thanks! You resolved the conversation after having done the work, but I like to leave the resolving to the reviewer so that it is clear what still needs to be rereviewed. But that's just a preference of course.